recipes/openexr/bld.bat

```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DILMBASE_PACKAGE_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
```

Comment:
Maybe we need `-DOPENEXR_BUILD_PYTHON_LIBS:BOOL=OFF` for the windows build.
Other options that are passed on unix and not windows are: `-DOPENEXR_NAMESPACE_VERSIONING:BOOL=OFF` and `-DOPENEXR_BUILD_STATIC:BOOL=ON`